test/common/init/init_fuzz_test.cc

```diff
@@ -0,0 +1,71 @@
+#include "common/common/assert.h"
+#include "common/init/manager_impl.h"
+#include "common/init/target_impl.h"
+#include "common/init/watcher_impl.h"
+
+#include "test/common/init/init_fuzz.pb.h"
+#include "test/fuzz/fuzz_runner.h"
+
+#include "absl/strings/str_cat.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Fuzz {
+namespace {
+using namespace Init;
+
+class InitDriver {
+public:
+  InitDriver(int manager_size, int target_size) {
```

Comment:
Prefer fixed size types in Envoy code style, unsigned where it makes sense like here.

Reply:
will do